src/js/utils/styles.js

```diff
@@ -41,17 +41,19 @@ export const edgeStyle = (
   responsiveBreakpoint,
   theme,
 ) => {
-  const breakpoint =
-    responsiveBreakpoint && theme.global.breakpoints[responsiveBreakpoint];
+  const breakpoint = getBreakpointStyle(theme, responsiveBreakpoint);
+  const metric = theme.global.edgeSize[data] || data;
```

Comment:
Behavior looks great. Two small naming suggestions would be to align the variable names with the concept of CSS value. Metric was a little bit confusing at first:
- metric --> value
- responsiveMetric --> responsiveValue

soft opinion.